firefox-ios/Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -1337,7 +1340,7 @@ class BrowserCoordinator: BaseCoordinator,
         }
 
         // If we successfully got a temp file URL, share it like a downloaded file
-        return .file(url: fileURL)
+        return .file(url: fileURL, remoteURL: shareType.wrappedURL)
```

Comment:
I was a bit confused at first since the wrappedURL is pulling from the `url` and not `remoteURL` in `ShareType`, but it seems this is in the right format so we use the existing shareType url for the new shared type.

Reply:
Ahhh okay that makes sense. Yeah I think if you add that, it will be a lot more helpful. Maybe adding this would help too? With a clarifying comment above?
`let tabRemoteUrl = shareType.wrappedURL`

Reply:
Oh yes much better!!! Thank you for restructuring it! 🥳